features/rhelah/atomic/sanity_test.feature

```diff
@@ -1,4 +1,4 @@
-@atomic @host_subscribed @ah_upgrade
+#@atomic @host_subscribed @ah_upgrade
```

Comment:
Instead of commenting this out, you can just remove the line.

If we need to introduce this again, we can always summon it from the git history.